server/src/api.ts

```diff
@@ -28,36 +23,37 @@ app.use(
 );
 app.use(express.json({ limit: '1mb' }));
 
-const port = config.httpPort;
-
 const server = http.createServer(app);
 
-const wss = new WebSocket.Server({ server });
-
-interface FindPlaysQuery extends PaginationParams {
+interface FindPlaysByPlayerParams {
   player_account_hash: string;
 }
 
-async function initAPI() {
+interface FindPlaysByRoundParams {
+  round_id: string;
+}
+
+async function main() {
   await AppDataSource.initialize();
 
   const playsRepository = new PlayRepository(AppDataSource);
   const roundsRepository = new RoundRepository(AppDataSource);
 
   const csprCloudClient = new CSPRCloudAPIClient(config.csprCloudApiUrl, config.csprCloudAccessKey);
 
-  app.get('/playsByPlayer', pagination(), async (req: Request<never, never, never, FindPlaysQuery>, res: Response) => {
-    const [plays, total] = await playsRepository.findByPlayer(req.query.player_account_hash, {
+  app.get('/players/:player_account_hash/plays', pagination(), async (req: Request<FindPlaysByPlayerParams, never, never, PaginationParams>, res: Response) => {
+    const [plays, total] = await playsRepository.getPaginatedPlays({
       limit: req.query.limit,
       offset: req.query.offset,
+      playerAccountHash: req.params.player_account_hash,
     });
 
     await csprCloudClient.withPublicKeys(plays);
 
     res.json({ data: plays, total });
   });
 
-  app.get('/plays', pagination(), async (req: Request<never, never, never, FindPlaysQuery>, res: Response) => {
+  app.get('/plays', pagination(), async (req: Request<never, never, never, PaginationParams>, res: Response) => {
```

Comment:
This one is not required anymore because we split it into two individual endpoints.